index.js

```diff
@@ -36,21 +39,19 @@ const deploy = async () => {
 			while (keepGoing) {
 				try {
 					const deploymentDetails = await axios.get(
-						`${base_url}/details/?instanceId=${instanceId}`,
+						`${base_url}/details?instanceId=${instanceId}`,
 						{ headers }
 					);
 					const state = deploymentDetails.data.metadata.status;
 					console.log(`Current state: ${state}`);
 
-					if (i > 100) {
-						console.error("Timing out after ~500 seconds.");
+					if (i > timeout * 10) {
```

Comment:
`i * 10` is the amount of seconds elapsed (roughly). However `timeout * 10` is not the amount of seconds in `timeout` minutes - I believe it is more. 

I think the right calculation is `i * 10 > timeout * 60` or more simply `i > timeout * 6`

Reply:
@TaylorChristie yeah you're right, bust have been late...